mainscreen.py

```diff
@@ -19,7 +19,7 @@
 from gettext import gettext as _
 import sys
 import json
-
+import pdb
```

Comment:
This was imported without being used, I guess it was for debugging.

Reply:
Oops, forgot :)